app/views/classified_listings/_form.html.erb

```diff
@@ -46,22 +45,22 @@
           <%= form.label "tags_list", "Tags" %>
           <%= form.text_field "tags_list", placeholder: "8 tags max, comma separated, no spaces or special characters" %>
         </div>
+      <% if @organizations %>
+      <div class="field">
+        <%= form.label "organization_id", "Post under an organization:" %>
+        <%= form.select :organization_id, "None" => "None" %>
+        <p><em>Posting on behalf of an organization spends the organization's credits.</em></p>
+      </div>
+      <% end %>
       </div>
     </div>
-    <%= javascript_pack_tag "listingForm", defer: true %>
+      <%= javascript_pack_tag "listingForm", defer: true %>
     <div class="field">
       <%= form.label "contact_via_connect", "Allow Users to Message Me Via In-App Chat (DEV Connect)" %>
       <%= form.check_box "contact_via_connect" %>
     </div>
-    <% if current_user.organization && current_user.org_admin %>
-      <div class="field">
-        <%= form.label "post_as_organization", "Post as #{current_user.organization.name} organization" %>
-        <%= form.check_box "post_as_organization" %>
-      </div>
-      <p><em>Posting on behalf of org spends org credits.</em></p>
-    <% end %>
     <div class="actions">
-      <% if @credits.size > 0 || (current_user.organization && current_user.organization.credits.where(spent: false).size > 0)%>
+      <% if @credits.size > 0 || (current_user.organization && current_user.organization.credits.where(spent: false).size > 0) %>
```

Comment:
So this actually will hit a run time error right now. I held off on fixing this because I was considering adding the counter_culture counting functionality in and using that solution for this.